source/Reviewers/howtocommit.rst

```diff
@@ -299,24 +299,29 @@ for all affected tests before you commit to the trunk.
         KGO files are stored in `$UMDIR/standard_jobs/kgo` and are installed there
         using a script.
 
-        1. Run the rose stem tasks that require a KGO update, plus any other testing required (see above) - if unsure run the `all` group.
+        1. Run the rose stem tasks that require a KGO update, plus any other testing required (see above) - if unsure run the `all,ex1a` group.
 
             .. code-block::
 
                 rose stem --group=all,ex1a --new
 
-        2. **As yourself**, in your merged Head of Trunk working copy move to ``admin/rose-stem``.
-        3. Run ``meto_update_kgo.sh``. This script will ask you to enter some details regarding the ticket.
+        2. You will need access to both your merged working copy and the SimSys_Scripts github repo. Run the script ``kgo_updates/meto_update_kgo.sh`` which is located in SimSys_Scripts.
+
+        3. The script will ask you to enter some details regarding the ticket.
 
           * Platforms: enter each platform which has a kgo change, lower case and space seperated, e.g. `spice xc40 ex1a`
+          * Path to your merged working copy - the script will check this exists and will fail if it can't be found.
+          * The file extension of the variables file. Currently this should be ``.rc`` for the UM and ``.cylc`` for lfricinputs.
```

Comment:
I haven't looked at the script itself yet (which isn't very helpful I know!) but this feels unintuitive to me... wouldn't it make more sense to ask if you're using cylc 7 or 8? Or is this because the UM rose-stem can be cylc 8 but in compatibility mode which means the files are still .rc's?

Reply:
Yeah it's a bit clunky and you're right - .rc and .cylc don't necessarily translate to cylc7 and cylc8. Obviously this is a stop gap until the UM fully updates to cylc8. Happy to change it if you can think of a better way?